rbi/sorbet/sorbet.rbi

```diff
@@ -276,6 +276,22 @@ class Sorbet::Private::Static::ENVClass
   end
   def keys; end
 
+  sig do
+    returns(Sorbet::Private::Static::ENVClass)
+  end
+  sig do
+    params(
+      hashes: T.any(Sorbet::Private::Static::ENVClass, T::Hash[String, T.nilable(String)])
```

Comment:
The docs don't mention the case where you can pass in another instance, should the `T.any` be changed to just a `T::Hash`?

Reply:
`merge!` will actually accept anything that is shaped like a `Hash` so you can call `ENV.merge!(ENV)` even if it doesn't make much sense.

I removed it 👍